mozilla/blurts-server
# How It Works Page - Mntor 2317

# References:

Jira: MNTOR-2317
Figma:  Multiple, see below



# Description
As a potential firefox monitor user i would like to learn how the service works so I can decide if i would like to use it.

Designs: (use long edit) https://www.figma.com/file/YqpVpUbSTzZMj9Jbp2po8Q/Mozilla-Monitor-Premium-LP?type=design&node-id=1173-23054&mode=design&t=uIz7n6wwuYHNOgwZ-4

# Screenshot (if applicable)
![Screenshot 2024-06-04 at 12 17 37 PM](https://github.com/mozilla/blurts-server/assets/31937894/d3fc2a38-10bc-4abb-94e3-65cb67419d00)

# How to test
Once dev server is running, navigate to [How It Works](http://localhost:6060/how-it-works) in your browser to see the page.  Links to the page are described in the [Jira ticket](https://mozilla-hub.atlassian.net/browse/MNTOR-2317), which also need to be manually reviewed.

The new page and updated links have 100% test coverage, so run tests locally.  Note:  A test for WelcomeToPlus.test.tsx
(checks the How-It-Works page link) is failing.  The test and the code being tested are almost identical to tests in LandingView.test.tsx, so the testing failing to find the element (a link) has been unresolved.  I have been advised to go ahead and do this PR in case someone else on the team is able to find a solution.

# Checklist (Definition of Done)
